### PR TITLE
Update: Initial c3 visualization extraction

### PR DESCRIPTION
**Pull request by jkusa**
Resolves #

## Proposed Changes

- move c3 visualization to `@yavin/c3` package
- rename `navi-vis-c3-chart` to `yavin-c3-chart`
- removed v2 visualization serializers
- removed c3 anomaly dataset feature
- removed jQuery from `navi-vis-c3-chart`
- converted request v1 mocks to v2

## License

I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

File: packages/c3/tests/unit/chart-builders/date-test.ts [hunk 1]
```diff
@@ -254,7 +254,7 @@ module('Unit | Chart Builders | Date Time', function (hooks) {
     });
     const data = DateChartBuilder.buildData(response, config, request);
 
-    assert.equal(data.series.length, 61, 'A data entry exists for each second in a minute');
+    assert.strictEqual(data.series.length, 61, 'A data entry exists for each second in a minute');
```
Comment (cythrawll) [inline packages/c3/tests/unit/chart-builders/date-test.ts:257]: why do these need to be strict equal?

Comment (jkusa) [reply packages/c3/tests/unit/chart-builders/date-test.ts:257]: lint rule :/

Comment (cythrawll) [reply packages/c3/tests/unit/chart-builders/date-test.ts:257]: ah must be a new one.

**Comment by cythrawll** [comment]
Question about the v2 serializers removal. Are we ready for this? all our viz models already v2?

File: packages/c3/tests/unit/chart-builders/date-test.ts (hunk 1 above)
Comment (jkusa) [reply packages/c3/tests/unit/chart-builders/date-test.ts:257]: yeah

**Comment by jkusa** [comment]
> Question about the v2 serializers removal. Are we ready for this? all our viz models already v2?

@cythrawll yes, I confirmed this with @kevinhinterlong in the beginning. All tests pass, but we should do another PR to clean up any remaining v1 logic.

**Review by cythrawll — APPROVED** [review]

**Review by kevinhinterlong — APPROVED** [review]

**Pull request merged**